I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 51, in script
    if oui:
  File "game/script.rpy", line 51, in <module>
    if oui:
NameError: name 'oui' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 51, in script
    if oui:
  File "C:\Users\a.riallot\Downloads\renpy-8.0.3-sdk\renpy\ast.py", line 2115, in execute
    if renpy.python.py_eval(condition):
  File "C:\Users\a.riallot\Downloads\renpy-8.0.3-sdk\renpy\python.py", line 1092, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\a.riallot\Downloads\renpy-8.0.3-sdk\renpy\python.py", line 1085, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 51, in <module>
    if oui:
NameError: name 'oui' is not defined

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.3.22090809
Workshop2_Test 1.0
Mon Feb  6 10:55:25 2023
